fabric-item-group-api-v1/src/client/java/net/fabricmc/fabric/impl/client/itemgroup/FabricCreativeGuiComponents.java

```diff
@@ -54,12 +56,12 @@ public void render(MatrixStack matrixStack, int mouseX, int mouseY, float float_
 			this.active = extensions.fabric_isButtonEnabled(type);
 
 			if (this.visible) {
-				int u = active && this.isHovered() ? 22 : 0;
-				int v = active ? 0 : 10;
+				int u = active && this.isHovered() ? 20 : 0;
+				int v = active ? ((this.isHovered() && programmer_art) ? 10 : 0) : 10;
```

Comment:
I am not a fan of doing the programmer art support like this. It means other texture packs will also have to create a programmer art version. I think its so minor it may be best to ingore it?